### DISCUSSION TITLE
Inline modifier with filtering functionality

### DISCUSSION THREAD

**Discussion by half5cat** [carbon-design-system/carbon#10384] [Help] [answered]
The documentation states the following:

> Note: Inline is only a modifier for dropdown and does not have filtering functionality.
> (Source: https://www.carbondesignsystem.com/components/dropdown/usage#modifiers )

However, in the react components I found [this example](https://react.carbondesignsystem.com/?path=/story/components-multiselect--filterable&aqs=chrome..69i57j69i58.964j0j1&ie=UTF-8&oq=iframe.html?id=components-multiselect--filterable&knob-MultiSelect%20ID%20(id)=carbon-multiselect-example&knob-Title%20(titleText)=Multiselect%20title&knob-Helper%20text%20(helperText)=This%20is%20helper%20text&knob-UI%20type%20(Only%20for%20`%3CMultiSelect%3E`)%20(type)=inline&knob-Dropdown%20direction%20(direction)=bottom&knob-Label%20(label)=MultiSelect%20Label&knob-Form%20validation%20UI%20content%20(invalidText)=Invalid%20Selection&knob-Warning%20state%20text%20(warnText)=Selecting%20more%20items%20may%20increase%20processing%20time&knob-Listbox%20menu%20icon%20translation%20IDs%20(for%20translateWithId%20callback)={%22close.menu%22:%22Close%20menu%22,%22open.menu%22:%22Open%20menu%22,%22clear.all%22:%22Clear%20all%22,%22clear.selection%22:%22Clear%20selection%22}&knob-Selection%20feedback=top-after-reopen&sourceid=chrome). There is no mention of an inline variant of the combobox, however. So, I would appreciate some clarification.

**Comment by tay1orjones** [comment, marked as answer]
Hi @half5cat, thanks for catching this - that blurb on the website should be updated to state that both Dropdown and MultiSelect support an `inline` variant.

**Reply by tay1orjones** [reply]
I've opened an issue to track getting this updated https://github.com/carbon-design-system/carbon-website/issues/2692

**Reply by half5cat** [reply]
Great! Thank you 👍
